### ISSUE TITLE
Loader.OnCompleteSignal wrong type

### ISSUE THREAD

**Issue by JackRontProdigy** [pixijs/pixijs#7663]
The type of `Loader.OnCompleteSignal` suggests that a single LoaderResource object is provided in the callback.  In fact, it is a collection of resources.  It also appears that the comment for it is incorrect.
```
/**
 * When the loader starts loading resources it dispatches this callback.
 * @param loader - The loader that has started loading resources.
 */
export type OnCompleteSignal = (loader: Loader, resource: LoaderResource) => void;
```

### Environment

- **`pixi.js` version**: 6.1.0

**Comment by bigtimebuddy** [comment]
Should probably be `(loader: Loader, resource: Record<string, LoaderResource>) => void;

**Comment by ivanpopelyshev** [comment]
I usually use `Dict` from our `utils`there

**Comment by bigtimebuddy** [comment]
Fixed with #7669
